ilastik/applets/counting/countingGui.py

```diff
@@ -783,7 +784,7 @@ def handleLabelSelectionChange(self):
         if self.topLevelOperatorView.MaxLabelValue.ready():
             enabled = True
             enabled &= self.topLevelOperatorView.MaxLabelValue.value >= 2
-            enabled &= numpy.all(numpy.asarray(self.topLevelOperatorView.CachedFeatureImages.meta.shape) > 0)
+            enabled &= bool(numpy.all(numpy.asarray(self.topLevelOperatorView.CachedFeatureImages.meta.shape) > 0))
```

Comment:
Maybe this is not the right PR to change this, but...
```python
value = self.topLevelOperatorView.MaxLabelValue.value
shape = self.topLevelOperatorView.CachedFeatureImages.meta.shape
enabled = value >= 2 and all(d > 0 for d in shape)
```

Reply:
ah, I'll add this in as it avoids the explicit cast to bool!